test/XRP/serializer.test.ts

```diff
@@ -2710,6 +2712,47 @@ describe('serializer', function (): void {
     assert.deepEqual(serialized, expectedJSON)
   })
 
+  it('serializes a TrustSet transaction with flags set', function (): void {
+    // GIVEN a transaction which represents the creation of a trust line linking two accounts, with flags.
+    const currency = 'USD'
+    const currencyIssuer = 'XVPcpSm47b1CZkf5AkKM9a84dQHe3m4sBhsrA4XtnBECTAc'
+    const transaction = xrpTestUtils.makeTrustSetTransaction(
```

Comment:
should we just update this method to optionally take flags as an argument?  Maybe not, I don't know, just an idea

Reply:
i did it, seems reasonable